lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -509,6 +506,18 @@ def main():
     except botocore.exceptions.NoCredentialsError as e:
         module.fail_json(msg=boto_exception(e))
 
+    # Wrap the cloudformation client methods that this module uses with
+    # automatic backoff / retry for throttling error codes
+    backoff_wrapper = AWSRetry.jittered_backoff(retries=10, delay=3, max_delay=30)
+    cfn.describe_stack_events = backoff_wrapper(cfn.describe_stack_events)
+    cfn.create_stack = backoff_wrapper(cfn.create_stack)
```

Comment:
I'm not super-enthused with the need to wrap calls like this, it seems difficult to keep in sync when we add calls to various cloudformation verbs. Thoughts?